graylog2-server/src/test/java/org/graylog/plugins/views/search/rest/SearchExecutorTest.java

```diff
@@ -71,8 +67,9 @@ public class SearchExecutorTest extends RestResourceBaseTest {
 
     private SearchExecutor searchExecutor;
 
-    @Before
-    public void setUp() throws Exception {
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
I've heard that you need to manually store and close object returned from openMocks(), more or less with the following code.

```
       private AutoCloseable closeable;

       @Before public void setUp() {
           closeable = MockitoAnnotations.openMocks(this);
       }

       @After public void releaseMocks() throws Exception {
           closeable.close();
       }
```

I'd say it is cleaner to add this one on the top of the whole class:
@ExtendWith(MockitoExtension.class)

Reply:
Good idea, the annotation seems like a good approach. It only needs one more configuration to avoid exceptions when not all mocks are used in every test:

```
@ExtendWith({MockitoExtension.class})
@MockitoSettings(strictness = Strictness.WARN)
```
Thanks for the hint!